build/BuildSteps.cs

```diff
@@ -107,17 +107,14 @@ public static void AddDistLib()
 
         public static void AddPythonWorker()
         {
-            var pythonWorkerPath = Path.Combine(Settings.OutputDir, "python-worker");
-            Directory.CreateDirectory(pythonWorkerPath);
-            using (var client = new WebClient())
-            {
-                client.DownloadFile(Settings.PythonWorkerUrl, Path.Combine(pythonWorkerPath, "worker.py"));
-                client.DownloadFile(Settings.PythonWorkerConfigUrl, Path.Combine(pythonWorkerPath, "worker.config.json"));
-            }
-
             foreach (var runtime in Settings.TargetRuntimes)
             {
-                FileHelpers.RecursiveCopy(pythonWorkerPath, Path.Combine(Settings.OutputDir, runtime, "workers", "python"));
+                // Python worker's dependencies are platform dependent and need to be copied accordingly
+                var pythonDir = Path.Combine(Settings.OutputDir, runtime, "workers", "python");
+                var allOsDirectories = Directory.GetDirectories(pythonDir);
+                var pythonPlatformSpecificWorker = Path.Combine(pythonDir, Settings.RuntimesToOS[runtime]);
```

Comment:
small nit, but I'd prefer if the path was all lowercase, so `Settings.RuntimeToOS[].ToLower()`

Reply:
@ahmelsayed, I think that would have to be updated in the nuget package itself (@maiqbal11). The runtimes `Windows, Linux, Mac` comes from the nuget package. 
After the build, they don't exist.

Reply:
That makes sense. Sorry I misread it.